### PR TITLE
overture: add new package

### PR DESCRIPTION
**Pull request by ysc3839**
Maintainer: me
Compile tested: x86-64 snapshot
Run tested: x86-64 snapshot

Description:
https://github.com/shawn1m/overture
Overture is a DNS server/forwarder/dispatcher written in Go.

There's an issue that there's relative path in default config file, but procd doesn't provide an option to change process current directory.
https://github.com/shawn1m/overture/blob/089a14d8b0804be61d9cad0755d2d30c40a1a85a/config.sample.json#L35-L38
Should I report this to bugs.openwrt.org?

### CODE REVIEW

File: net/overture/Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=overture
+PKG_VERSION:=1.6-rc1
```
Comment (BKPepe) [inline net/overture/Makefile]: Is there any reason why you want to add this version, which is in RC?

Comment (ysc3839) [reply net/overture/Makefile]: Because I think it's the latest. I will change to latest stable version.

**Comment by ysc3839** [comment]
Forget to change `PKG_HASH`, sorry for my mistake.

**Comment by ysc3839** [comment]
Build failed with v1.5, waiting upstream for next stable release.

File: net/overture/files/overture.init [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh /etc/rc.common
+
+START=90
```
Comment (diizzyy) [inline net/overture/files/overture.init]: Any reason why this is 90 instead of 99?

Comment (ysc3839) [reply net/overture/files/overture.init]: Just think this is a DNS related service, maybe need to start earlier.
dnscrypt-proxy is 30.
https://github.com/openwrt/packages/blob/7fb0a94fb5af8f7a52c58a9472d2bc2a2a70ac2e/net/dnscrypt-proxy/files/dnscrypt-proxy.init#L3

**Comment by ysc3839** [comment]
Updated to 1.6. Compile & run tested on x86-64 snapshot.

**Comment by ysc3839** [comment]
Ping @BKPepe @diizzyy

**Pull request merged**